lib/ansible/galaxy/collection/concrete_artifact_manager.py

```diff
@@ -605,6 +625,25 @@ def _normalize_galaxy_yml_manifest(
     return galaxy_yml
 
 
+@cache
+def _get_runtime_from_dir(b_path: bytes) -> t.Any:
```

Comment:
Would `object` be enough?
```suggestion
def _get_runtime_from_dir(b_path: bytes) -> object:
```

Reply:
Fixed.